cpp/ql/lib/codeql/change-notes/2021-11-01-isFromSystemMacroDefinition.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Could you add a readme or some other file that will stick around in each of these directories to explain how they are supposed to work?

Or we at least need to have a good explanation somewhere that is easy to find.